### DISCUSSION TITLE
Preview PDF chapter in quarto book

### DISCUSSION THREAD

**Discussion by Ifitrains** [quarto-dev/quarto-cli#7762] [Q&A] [answered]
### Description

Hi,

is it someone possible to only render one chapter from a quarto book (output format pdf)?

**Comment by mcanouil** [comment]
Yes, don't add the other chapters in the YAML (or maybe use `render` to define what should be rendered).
See the documentation.

**Reply by memeplex** [reply]
I've asked [this similar but more detailed question](https://github.com/quarto-dev/quarto-cli/discussions/8895) because I'm not able to get the desired result using the render option. If you prefer to keep the discussion here I can close it.

**Reply by danieltomasz** [reply]
@memeplex
 1) rename your project config to `_quarto-book.yml` (or other name) , copy the same project config as `_quarto.ym`' but remove the mentions of chapters, and you will be able to render your file withyour command; then you can run you whole book as `quarto render --profile book` (you can substitute book with chosen profile name); 
 2) another way - maybe simpler =   just create a second profile `_quarto-chapter1.yml` ( with only one chapter inlcued  and then run `quarto render --profile chapter1`

**Comment by danieltomasz** [comment, marked as answer]
@Ifitrains I have a workaround with quarto profiles, I dont want to constantly switch between chapters in yaml, so I created an additional profile which I can use to include only the  chapter I work on this moment

**Reply by mcanouil** [reply]
Indeed, this works too.
